src/cbor_decode.c

```diff
@@ -0,0 +1,390 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <string.h>
+#include "cbor_decode.h"
+
+static uint32_t additional_len(uint8_t additional)
+{
+	switch (additional) {
+	case 24: return 1;
```

Comment:
Torn. I like the actual values, but i would like a define as these are matching values

could possible be done only using a comment describing this list